### PR TITLE
Add `OWC 2024` news post, wiki article

### PR DESCRIPTION
**Pull request by LeoFLT**
## Pre-merge checklist

- [x] Graphics
- [x] Tournament listing (depends on having graphics)

Please redate as appropriate (so as to not conflict with MWC 4K 2024's Finals recap as much as possible). The files have been linked as if they were there already. This will of course result in a CI error but the review process doesn't hinge on it so here it is.

### CODE REVIEW

File: news/2024/2024-09-26-osu-world-cup-2024-registrations-now-open.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+date: 2024-09-26 12:00:00 +0000
+---
+
+The osu! World Cup is back for its 15th edition, but now with a slight twist our beloved country-based circle clicking festival. Read on to find out why!
+
+![](/wiki/Tournaments/OWC/2024/img/owc2024-banner.jpg)
+
+Hot on the heels of our very own [osu!mania 4K World Cup Finals week](https://osu.ppy.sh/home/news/2024-09-25-osumania-4k-world-cup-2024-finals-recap) comes OWC 2024, with changes to the rank range and pick/ban phase procedures.
+
+For this year's edition, the tournament structure remains largely the same, from the seeding, to the bracket format and even mappool. We do, however, have two major changes, the first of which has to do with the rank range. For the 2024 osu! World Cup, we have raised the minimum rank required to sign up to **#10,000 in the osu! global ranking**. We believe that the average skill on the this new rank range has risen considerably over the last few years, and we're excited to see the new team building possibilities this brings, especially for players focusing on more niche skill sets. **There are no plans to apply this new rank range to other modes currently.**
```
Comment (Walavouchey) [inline news/2024/2024-09-26-osu-world-cup-2024-registrations-now-open.md:13]: long

```suggestion
For this year's edition, the tournament structure remains largely the same, from the seeding, to the bracket format and even mappool. We do, however, have two major changes, the first of which has to do with the rank range. For the 2024 osu! World Cup, we have raised the minimum rank required to sign up to **#10,000 in the osu! global ranking**.

We believe that the average skill on the this new rank range has risen considerably over the last few years, and we're excited to see the new team building possibilities this brings, especially for players focusing on more niche skill sets. **There are no plans to apply this new rank range to other modes currently.**
```

File: news/2024/2024-09-26-osu-world-cup-2024-registrations-now-open.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+The second change has all to do with how our mappools are structured, and how the pick and ban system interacts with it. We've opted to lift the restrictions on double banning beatmaps from the Hidden, Hard Rock, Double time, and Free Mod pool slots. In the words of our ever reliable mappooler and #mappool-feedback PR specialist [chiv](https://osu.ppy.sh/users/6701656):
```
Comment (Walavouchey) [inline news/2024/2024-09-26-osu-world-cup-2024-registrations-now-open.md:15]:
```suggestion
The second change has all to do with how our mappools are structured, and how the pick and ban system interacts with it. We've opted to lift the restrictions on double banning beatmaps from the Hidden, Hard Rock, Double time, and Free Mod pool slots. In the words of our ever reliable mappooler and `#mappool-feedback` PR specialist [chiv](https://osu.ppy.sh/users/6701656):
```

File: news/2024/2024-09-26-osu-world-cup-2024-registrations-now-open.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+> "The current restriction on banning two maps of the same category is not compatible with modern mappools, which often have considerable fluctuations in skill set distribution across the different slots."
```
Comment (Walavouchey) [inline news/2024/2024-09-26-osu-world-cup-2024-registrations-now-open.md:17]:
```suggestion
> The current restriction on banning two maps of the same category is not compatible with modern mappools, which often have considerable fluctuations in skill set distribution across the different slots.
```

File: news/2024/2024-09-26-osu-world-cup-2024-registrations-now-open.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+In short: **slot pooling is no more.**
+
+Refer to the **[osu! World Cup 2024 wiki article](/wiki/Tournaments/OWC/2024)** for more information regarding the tournament. You can also discuss this event in the **[official discussion thread](https://osu.ppy.sh/community/forums/topics/1982268)** or the public **[community server](https://discord.gg/0Vxo9AsejDkGlk3H)**.
```
Comment (Walavouchey) [inline news/2024/2024-09-26-osu-world-cup-2024-registrations-now-open.md:21]:
```suggestion
Refer to the **[osu! World Cup 2024 wiki article](/wiki/Tournaments/OWC/2024)** for more information regarding the tournament. You can also discuss this event in the **[official discussion thread](https://osu.ppy.sh/community/forums/topics/1982268)** or the public **[community server](https://discord.com/invite/0Vxo9AsejDkGlk3H)**.
```

File: news/2024/2024-09-26-osu-world-cup-2024-registrations-now-open.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+| --: | :-- |
+| Registration phase | Sep 26 (12:00 UTC) – Oct 10 (23:59 UTC) |
+| Qualifier showcase | Oct 20 (14:00 UTC) |
+| Qualifier stage | Oct 26 & Oct 27 |
+| Round of 32 | Nov 02 & Nov 03 |
+| Round of 16 | Nov 09 & Nov 10 |
+| Quarterfinals | Nov 16 & Nov 17 |
+| Semifinals | Nov 23 & Nov 24 |
+| Finals | Nov 30 & Dec 01 |
+| Grand Finals | Dec 07 & Dec 08 |
```
Comment (Walavouchey) [inline news/2024/2024-09-26-osu-world-cup-2024-registrations-now-open.md:35]:
```suggestion
| Round of 32 | Nov 2 & Nov 3 |
| Round of 16 | Nov 9 & Nov 10 |
| Quarterfinals | Nov 16 & Nov 17 |
| Semifinals | Nov 23 & Nov 24 |
| Finals | Nov 30 & Dec 1 |
| Grand Finals | Dec 7 & Dec 8 |
```

File: wiki/Tournaments/OWC/2024/en.md [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+### Qualifier instructions
+
+1. In the Qualifier stage, all teams will play a specific pool designed by the mappool selectors.
+2. The mappool will contain 4 brackets: No Mod, [Hidden](/wiki/Gameplay/Game_modifier/Hidden), [Hard Rock](/wiki/Gameplay/Game_modifier/Hard_Rock), and [Double Time](/wiki/Gameplay/Game_modifier/Double_Time). There will be no Free Mods or tiebreakers for this stage.
+3. The Qualifier pool will feature 11 beatmaps, organised as such:
+   - No Mod: 4 beatmaps
+   - Hidden: 2 beatmaps
+   - Hard Rock: 2 beatmaps
+   - Double Time: 3 beatmaps
+4. Teams will be asked to play the mappool twice at a time of their choosing, considering timeslot availability and any applicable deadlines. **Only the team's best playthrough for each beatmap, that is, the team score grouped per beatmap as seen on the multiplayer lobby link, will be used for seeding**.
```
Comment (Walavouchey) [inline wiki/Tournaments/OWC/2024/en.md:132]:
```suggestion
4. Teams will be asked to play the mappool twice at a time of their choosing, considering time slot availability and any applicable deadlines. **Only the team's best playthrough for each beatmap, that is, the team score grouped per beatmap as seen on the multiplayer lobby link, will be used for seeding**.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

**Review by Walavouchey — APPROVED** [review]

File: news/2024/2024-09-26-osu-world-cup-2024-registrations-now-open.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+The osu! World Cup is back for its 15th edition, but now with a slight twist our beloved country-based circle clicking festival. Read on to find out why!
+
+![](/wiki/Tournaments/OWC/2024/img/owc2024-banner.jpg)
+
+Hot on the heels of our very own [osu!mania 4K World Cup Finals week](https://osu.ppy.sh/home/news/2024-09-25-osumania-4k-world-cup-2024-finals-recap) comes OWC 2024, with changes to the rank range and pick/ban phase procedures.
+
+For this year's edition, the tournament structure remains largely the same, from the seeding, to the bracket format and even mappool. We do, however, have two major changes, the first of which has to do with the rank range. For the 2024 osu! World Cup, we have raised the minimum rank required to sign up to **#10,000 in the osu! global ranking**.
+
+We believe that the average skill on the this new rank range has risen considerably over the last few years, and we're excited to see the new team building possibilities this brings, especially for players focusing on more niche skill sets. **There are no plans to apply this new rank range to other modes currently.**
```
Comment (Y2nn1c) [inline news/2024/2024-09-26-osu-world-cup-2024-registrations-now-open.md:15]:
```suggestion
We believe that the average skill on the this new rank range has risen considerably over the last few years, and we're excited to see the new team building possibilities this brings, especially for players focusing on more niche skill sets. **There are currently no plans to apply this new rank range to other modes as well.**
```

File: news/2024/2024-09-26-osu-world-cup-2024-registrations-now-open.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+
+The second change has all to do with how our mappools are structured, and how the pick and ban system interacts with it. We've opted to lift the restrictions on double banning beatmaps from the Hidden, Hard Rock, Double time, and Free Mod pool slots. In the words of our ever reliable mappooler and `#mappool-feedback` PR specialist [chiv](https://osu.ppy.sh/users/6701656):
```
Comment (Joehuu) [inline news/2024/2024-09-26-osu-world-cup-2024-registrations-now-open.md:17]:
```suggestion
The second change has all to do with how our mappools are structured, and how the pick and ban system interacts with it. We've opted to lift the restrictions on double banning beatmaps from the Hidden, Hard Rock, Double Time, and Free Mod pool slots. In the words of our ever reliable mappooler and `#mappool-feedback` PR specialist [chiv](https://osu.ppy.sh/users/6701656):
```

File: news/2024/2024-09-26-osu-world-cup-2024-registrations-now-open.md (hunk 7 above)
Comment (Joehuu) [reply news/2024/2024-09-26-osu-world-cup-2024-registrations-now-open.md:15]:
```suggestion
We believe that the average skill on this new rank range has risen considerably over the last few years, and we're excited to see the new team building possibilities this brings, especially for players focusing on more niche skill sets. **There are no plans to apply this new rank range to other modes currently.**
```

File: news/2024/2024-09-26-osu-world-cup-2024-registrations-now-open.md (hunk 8 above)
Comment (Y2nn1c) [inline news/2024/2024-09-26-osu-world-cup-2024-registrations-now-open.md:17]:
```suggestion
The second change is all about the structure of our mappools, and how the pick and ban system interacts with it. We've opted to lift the restrictions on double banning beatmaps from the Hidden, Hard Rock, Double Time, and Free Mod pool slots. In the words of our ever reliable mappooler and `#mappool-feedback` PR specialist [chiv](https://osu.ppy.sh/users/6701656):
```

File: news/2024/2024-09-26-osu-world-cup-2024-registrations-now-open.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+
+> The current restriction on banning two maps of the same category is not compatible with modern mappools, which often have considerable fluctuations in skill set distribution across the different slots.
```
Comment (Y2nn1c) [inline news/2024/2024-09-26-osu-world-cup-2024-registrations-now-open.md:19]:
```suggestion
> The current restriction on banning two maps of the same category is not compatible with modern mappools, which often feature considerable fluctuations in skill set distribution across the different slots.
```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**